storage/src/snapshot_cache.rs

```diff
@@ -0,0 +1,268 @@
+use crate::Snapshot;
+use std::collections::VecDeque;
+
+use anyhow::anyhow;
+
+/// The number of snapshots kept in the [`SnapshotCache`].
+pub const DEFAULT_SNAPSHOT_CACHE_SIZE: usize = 10;
```

Comment:
Since the actual size is set in the constructor, it's better not to have a constant here, because having the constant as part of the public API suggests that it does something, when in fact it's not used at all internally, it's just a value that an API consumer *could* put into the constructor.

Reply:
Thanks, this makes sense